tests/e2e/ctl_v3_snapshot_test.go

```diff
@@ -136,6 +134,7 @@ func ctlV3SnapshotSave(cx ctlCtx, fpath string) error {
 }
 
 func getSnapshotStatus(cx ctlCtx, fpath string) (snapshot.Status, error) {
+	cx.etcdutl = true
```

Comment:
Would be good to make etcdutl default for all removed commands and remove `withEtcdutl()` and ctlCtx.etcdutl. Can be either in the same or separate PR.

Reply:
Other Fixes expect this. I will commit another PR to fix.

Reply:
sg